package.json

```diff
@@ -29,10 +29,8 @@
 		"wpcom": [
 			"extends @wordpress/browserslist-config"
 		],
-		"defaults": [
-			"extends @wordpress/browserslist-config"
-		],
-		"evergreen": [
+		"calypso": [
```

Comment:
We need to be very mindful of removing the `defaults` target. Doing so basically means that any `browserslist`-aware build tool that is run without a `BROWSERSLIST_ENV` env variable set will use the `browserslist` standard defaults, and not any of our own targets. This could lead to some unexpected results.

Reply:
From the docs, Browserlist' default is actually not `defaults` but `production` (https://github.com/browserslist/browserslist#configuring-for-different-environments).

So this was already happening (some wpcom build tools were not setting `BROWSERLIST_ENV`). We were lucky that browserlists' defaults target the same browsers that our `defaults` environment.

Reply:
I don't think that's true, based on my experience working with and on `browserslist`. While `production` is definitely preferred over `defaults`, the latter is very much still used.

To make sure, I set up a small test. With no `browserslist` config:

```
$ npx browserslist
and_chr 89
and_ff 86
and_qq 10.4
and_uc 12.12
android 89
baidu 7.12
chrome 89
chrome 88
chrome 87
edge 89
edge 88
firefox 86
firefox 85
firefox 78
ie 11
ios_saf 14.0-14.5
ios_saf 13.4-13.7
kaios 2.5
op_mini all
op_mob 62
opera 73
opera 72
safari 14
safari 13.1
samsung 13.0
samsung 12.0
```

That's the default `browserslist` config, as expected.

With a `defaults` entry in `package.json` set to `last 2 chrome versions`:

```
$ npx browserslist
chrome 89
chrome 88
```

We've successfully overridden `browserlist`'s defaults with our own.

With both the `defaults` entry and a new `production` entry set to `last 2 safari versions`:

```
$ npx browserslist
safari 14
safari 13.1
```

The `production` config is preferred, when both `production` and `defaults` are available.

Reply:
Interesting! Good catch, I'll leave `defaults` then.

Reply:
Haha, I think I see where the confusion is.

> If none of them is declared, Browserslist will firstly look for `production` queries and then use defaults.

I read that as "If none of them is declared, Browserslist will firstly look for `production` queries and then use THE defaults." (meaning if the environment `production` is not defined, then it will use the default list of browsers from Browserslist, defined as `> 0.5%, last 2 versions, Firefox ESR, not dead.` in https://github.com/browserslist/browserslist#queries)

But it actually means: "If none of them is declared, Browserslist will firstly look for `production` queries and then use `defaults`."

The difference that a couple of `` can make :)

Reply:
Ah, that explains it! :) It really goes to show how careful one has to be when writing docs!